docs: update jina embeddings integration

bwanglzu opened the pull request:
Update documentation of jina embeddings:

1. Created a detailed model list with tech summary.
2. add description of task adapters of `jina-embeddings-v3` and how to use them.
3. add short description of MRL performance to give user a hint how to balance between storage and performance.
4. updated code snnipt.

CatStark commented on qdrant-landing/content/documentation/embeddings/jina-embeddings.md:
```diff
@@ -8,21 +8,58 @@ aliases:
+|  jina-embeddings-v3  |  1024 | Multilingual (89 languages)  |  Yes  | 8192 |
+|  jina-embeddings-v2-base-en |  768 |  English |  No | 8192  | 
+|  jina-embeddings-v2-base-de |  768 |  German & English |  No  |  8192 | 
+|  jina-embeddings-v2-base-de |  768 |  Spanish & English |  No  |  8192 | 
```
```suggestion
|  jina-embeddings-v2-base-es |  768 |  Spanish & English |  No  |  8192 | 
```

Anush008 commented on qdrant-landing/content/documentation/embeddings/jina-embeddings.md:
```diff
@@ -8,21 +8,58 @@ aliases:
 # Jina Embeddings
 
-Qdrant can also easily work with [Jina embeddings](https://jina.ai/embeddings/) which allow for model input lengths of up to 8192 tokens.
+Qdrant has been integrated with [Jina AI](https://jina.ai/). You need an API key from [Jina AI](https://jina.ai/embeddings/) to get embeddings.
```
```suggestion
Qdrant is compatible with [Jina AI](https://jina.ai/) embeddings. You need an API key from [Jina AI](https://jina.ai/embeddings/) to get embeddings.
```

Anush008 commented on qdrant-landing/content/documentation/embeddings/jina-embeddings.md:
```diff
@@ -8,21 +8,58 @@ aliases:
+|  jina-embeddings-v2-base-es |  768 |  Spanish & English |  No  |  8192 | 
+|  jina-embeddings-v2-base-zh | 768  |  Chinese & English |  No  |  8192 | 
+
+> We recommend using `jina-embeddings-v3` as it is the latest and most performant embedding model released by Jina AI.
```
```suggestion
> Jina recommend using `jina-embeddings-v3` as it is the latest and most performant embedding model released by Jina AI.
```

Anush008 commented on qdrant-landing/content/documentation/embeddings/jina-embeddings.md:
```diff
@@ -8,21 +8,58 @@ aliases:
+import qdrant_client
 from qdrant_client.models import Distance, VectorParams, Batch
 
 # Provide Jina API key and choose one of the available models.
 # You can get a free trial key here: https://jina.ai/embeddings/
```
Probably mention the free trial in the texts above instead of this comment.

Anush008 commented on qdrant-landing/content/documentation/embeddings/jina-embeddings.md:
```diff
@@ -8,21 +8,58 @@ aliases:
+|  jina-embeddings-v2-base-es |  768 |  Spanish & English |  No  |  8192 | 
+|  jina-embeddings-v2-base-zh | 768  |  Chinese & English |  No  |  8192 | 
+
+> Jina recommend using `jina-embeddings-v3` as it is the latest and most performant embedding model released by Jina AI.
```
```suggestion
> Jina recommends using `jina-embeddings-v3` as it is the latest and most performant embedding model released by Jina AI.
```

bwanglzu commented:
@Anush008 thanks for your review, please do not merge right now we might make a small change to api schema

bwanglzu commented:
@Anush008 PR updated, two small changes since last review:

1. rename `task_type` to `task` to adopt API change.
2. remove "free api key" from the code sample

Anush008 commented on qdrant-landing/content/documentation/embeddings/jina-embeddings.md:
```diff
@@ -8,21 +8,57 @@ aliases:
+|  Average Retrieval Performance (nDCG@10)   |   52.54     | 58.54 |    61.64    | 62.72 | 63.16  | 63.3  |   63.35    | 
+
+
+## Guide
```
```suggestion
## Example
```

Anush008 commented on qdrant-landing/content/documentation/embeddings/jina-embeddings.md:
```diff
@@ -8,21 +8,57 @@ aliases:
-Qdrant can also easily work with [Jina embeddings](https://jina.ai/embeddings/) which allow for model input lengths of up to 8192 tokens.
+Qdrant is compatible with [Jina AI](https://jina.ai/) embeddings. You can get a free trial key from [Jina Embeddings](https://jina.ai/embeddings/) to get embeddings.
+
+For Qdrant users, use code **QDRANT** for a 10% discount on Jina AI APIs.
```
```suggestion
Qdrant users can receive a 10% discount on Jina AI APIs by using the code **QDRANT**.
```

Anush008 reviewed (approved):
Thank you.
